.env

```
@@ -0,0 +1,40 @@
+Domain=task.labsacme.com
+WEB_URL=${Domain}
+PGHOST=plane-db
+PGDATABASE=plane
+POSTGRES_USER=colton84
+POSTGRES_PASSWORD=hk3nhmqkydastxnkezqdelfwefzylxvm
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)